multicluster/controllers/multicluster/member/serviceexport_controller.go

```diff
@@ -626,10 +665,106 @@ func (r *ServiceExportReconciler) updateOrCreateResourceExport(resName string,
 	return nil
 }
 
+// getSubsetsFromEndpointSlice will check all ready Endpoints from any matched EndpointSlice
+// The purpose is to simplify Endpoints change detection since we already mapped the EndpointSlice
+// change event into a ServiceExport event. It will be hard to track individual EndpointSlices change
```

Comment:
Say: In future, we should change to track and export individual EndpointSlices, rather than merge them to one Endpoints.

I do not understand "It will be hard to track individual EndpointSlices change after mapping", "all ready Endpoints from any matched EndpointSlice". If you do not want to rephase, maybe remove them.

Reply:
Refined. thanks.